man/fi_endpoint.3.md

```diff
@@ -714,50 +792,186 @@ order values are:
+which message data is sent or received by the transport layer.  Message
+ordering requires matching ordering semantics on the transmitting side of a
+data transfer operation in order to guarantee that ordering is met.
+
+*FI_ORDER_NONE*
+: No ordering is specified.  This value may be used as input in order
+  to obtain the default message order supported by the provider.
+
+*FI_ORDER_RAR*
+: Read after read.  If set, RMA and atomic read operations are
+  processed in the order that they were transmitted relative to other
+  RMA and atomic read operations.  If not set, RMA and atomic reads
+  may be processed out of order from their submission.
+
+*FI_ORDER_RAW*
+: Read after write.  If set, RMA and atomic read operations are
+  processed in the order that they were transmitted relative to RMA
+  and atomic write operations.  If not set, RMA and atomic reads may be
+  processed ahead RMA and atomic writes.
+
+*FI_ORDER_RAS*
+: Read after send.  If set, RMA and atomic read operations are
+  processed in the order that they were transmitted relative to message send
+  operations, including tagged sends.  If not set, RMA and atomic
+  reads may be processed ahead of sends.
+
+*FI_ORDER_WAR*
+: Write after read.  If set, RMA and atomic write operations are
+  processed in the order that they were transmitted relative to RMA and atomic
+  read operations.  If not set, RMA and atomic writes may be processed
+  ahead of RMA and atomic reads.
+
+*FI_ORDER_WAW*
+: Write after write.  If set, RMA and atomic write operations are
+  processed in the order that they were transmitted relative to other RMA
+  and atomic write operations.  If not set, RMA and atomic writes may be
+  processed out of order from their submission.
+
+*FI_ORDER_WAS*
+: Write after send.  If set, RMA and atomic write operations are
+  processed in the order that they were transmitted relative to message send
+  operations, including tagged sends.  If not set, RMA and atomic
+  writes may be processed ahead of sends.
+
+*FI_ORDER_SAR*
+: Send after read.  If set, message send operations, including tagged
+  sends, are processed in the order that they were transmitted relative to RMA
+  and atomic read operations.  If not set, message sends may be processed ahead
+  of RMA and atomic reads.
+
+*FI_ORDER_SAW*
+: Send after write.  If set, message send operations, including tagged
+  sends, are processed in the order that they were transmitted relative to RMA
+  and atomic write operations.  If not set, message sends may be processed ahead
+  of RMA and atomic writes.
+
+*FI_ORDER_SAS*
+: Send after send.  If set, message send operations, including tagged
+  sends, are processed in the order that they were transmitted relative to other
+  message send.  If not set, message sends may be processed out of
+  order from their submission.
+
+## comp_order - Completion Ordering
+
+For a description of message ordering, see the msg_order field in
+the _Transmit Context Attribute_ section.
```

Comment:
I was referring to the paragraph underneath the completion ordering heading.  GitHub just gave lots of irrelevant context.

On March 31, 2015 6:48:06 PM EDT, Sean Hefty <notifications@github.com> wrote:
>> +*FI_ORDER_SAW*
>> +: Send after write.  If set, message send operations, including
>tagged
>> +  sends, are processed in the order that they were transmitted
>relative to RMA
>> +  and atomic write operations.  If not set, message sends may be
>processed ahead
>> +  of RMA and atomic writes.
>> +
>> +*FI_ORDER_SAS*
>> +: Send after send.  If set, message send operations, including
>tagged
>> +  sends, are processed in the order that they were transmitted
>relative to other
>> +  message send.  If not set, message sends may be processed out of
>> +  order from their submission.
>> +
>> +## comp_order - Completion Ordering
>> +
>> +For a description of message ordering, see the msg_order field in
>> +the _Transmit Context Attribute_ section.
>
>I'm assuming that you're referring to the order here.  The format
>matches the syntax used in other places in the file, where the field
>name comes first, followed by the description.  I can go either way
>with that, but all of the man pages would need to be updated for
>consistency.
>
>Btw, thanks for pointing out that I left this series unmerged.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ofiwg/libfabric/pull/850/files#r27530884

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.